cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -2433,19 +2433,21 @@ def test_obs_added_tissue_type_label__unknown(self, validator_with_adata):
         obs.at["Y", "tissue_type"] = "cell culture"  # Already set in example data, just setting explicitly here
         obs.at["Y", "tissue_ontology_term_id"] = "unknown"  # Testing this term case
         validator_with_adata.validate_adata()  # Validate
-        AnnDataLabelAppender(validator_with_adata)._add_labels()  # Annotate
+        labeler = AnnDataLabelAppender(validator_with_adata)
```

Comment:
a new adata container is being created in the `AnnDataLabelAppender`, so we need to check that the `AnnDataLabelAppender.adata` is being updated when we call `_add_labels`